nixd/lspserver/include/lspserver/LSPServer.h

```diff
@@ -47,7 +49,8 @@ class LSPServer : public MessageHandler {
   void callMethod(llvm::StringRef Method, llvm::json::Value Params,
                   Callback<llvm::json::Value> CB, OutboundPort *O) {
     llvm::json::Value ID(bindReply(std::move(CB)));
-    log("--> call {0}({1})", Method, ID.getAsInteger());
```

Comment:
to consider introducing a `maybe_log` or setting log level such that `log` becomes a no-op.